test/controllers/cookie_authentications_controller_test.rb

```diff
@@ -51,7 +51,7 @@ class CookieAuthenticationsControllerTest < ActionDispatch::IntegrationTest
 
     login(users(:mallory_archer))
 
-    Timecop.travel 15.minutes.from_now
+    Timecop.travel 30.seconds.from_now
```

Comment:
I lowered access token expiration from 60 minutes (an hour) down to 5 minutes.  Caused tests to fail!  Also gratifying.